docs/components/popover/README.md

```diff
@@ -132,16 +132,19 @@ CdrPopover is a wrapper component that accepts a trigger element and popover con
 
 ## Accessibility
 
-Ensure that usage of this component complies with accessibility guidelines:
+To ensure your usage of this component complies with accessibility guidelines you are responsible for the following:
 
 - Set an `id` property on the CdrPopover. The component will automatically link that `id` to the trigger element.
 - Content passed in to the `trigger` slot must be an actionable element such as a button.
-- TODO: Popover content should ...
+- Popovers can contain complex content structures such as headers, lists, and actionable items that provide related additional content to the section of the triggering element.
+
 
 This component complies with WCAG guidelines by:
 
 - Adds click handler to trigger element that manages popover state.
 - When opened, moves focus to the first actionable element inside the popover.
+  - If a dialog is limited to interactions that either provide additional information or continue processing, it may be advisable to set focus to the element that is likely to be most frequently used, such as an OK or Continue button.
```

Comment:
right now the popover is coded to set focus to the first tabbable element in the popup. 

The frequently used button example would work as long as that button is the first actionable element in the popover. (the close button is absolute positioned so its always the last actionable element in the document order)

The tabindex=-1 case would never work since its not tabbable. Also the popover/tooltip have no scroll logic built into them so its kind of a moot point.

Not sure how common that second use case is or if there's another way to sovle it

Reply:
yea if the popover will never need to scroll we can omit that - these are handy more for deviations from our out of the box solution - maybe we should create some other section for the general patterns - or move them to confluence...

Reply:
yeah, i think it makes sense to update line 146 to note that we auto-focus the first tabbable element, and move line 147 to a new general a11y section below